### PR TITLE
Consume terra-dev-site 1.0

### PR DESCRIPTION
**Pull request by mjhenkes**
Delete site, action header, error view and no data view

### Summary
Move examples.
Delete site, action header, error view and no data view

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-const config = require('./packages/terra-clinical-site/postcss.config');
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (emilyrohrbough) [inline postcss.config.js]: The default config handle this, so this can be removed

Comment (mjhenkes) [reply postcss.config.js]: the whole file?

Comment (emilyrohrbough) [reply postcss.config.js]: Yes the postcss configuration is defined in terra-dev-site's webpack config

**Comment by emilyrohrbough** [comment]
We can remove the i18n section on the repo readme at the time since there no internationalized components in terra-clincial

**Comment by mjhenkes** [comment]
@emilyrohrbough onset picker is internationalized, but that section on the read me is dumb and gets outdated very easily. I'll remove it.

**Comment by emilyrohrbough** [comment]
Ah, forgot about that one.

**Pull request merged**